org.eclipse.scout.rt.api.data/src/main/java/org/eclipse/scout/rt/api/data/ApiExposeHelper.java

```diff
@@ -26,28 +27,49 @@ public class ApiExposeHelper {
   public static final String OBJECT_TYPE_ATTRIBUTE_NAME = "objectType";
 
   /**
-   * @return if the given instance has the {@link ApiExposed} annotation set (directly or on one of its super classes).
+   * @return if the given instance has the {@link ApiExposed} annotation with {@link ApiExposed#value()} = {@code true} set (directly or on one of its super classes).
    */
-  public boolean hasApiExposedAnnotation(Object instance) {
+  public boolean isApiExposed(Object instance) {
     if (instance == null) {
       return false;
     }
-    return hasApiExposedAnnotation(instance.getClass());
+    return isApiExposed(instance.getClass());
   }
 
   /**
-   * @return if the given {@link IBean} has the {@link ApiExposed} annotation set (directly or on one of its super
+   * @return if the given {@link IBean} has the {@link ApiExposed} annotation with {@link ApiExposed#value()} = {@code true} set (directly or on one of its super
    * classes).
    */
-  public boolean hasApiExposedAnnotation(IBean<?> bean) {
-    return getAnnotation(bean, ApiExposed.class) != null;
+  public boolean isApiExposed(IBean<?> bean) {
+    ApiExposed ann = getAnnotation(bean, ApiExposed.class);
+    return ann != null && ann.value();
   }
 
   /**
-   * @return if the given class has the {@link ApiExposed} annotation set (directly or on one of its super classes).
+   * @return if the given class has the {@link ApiExposed} annotation with {@link ApiExposed#value()} = {@code true}  set (directly or on one of its super classes).
    */
-  public boolean hasApiExposedAnnotation(Class<?> clazz) {
-    return getAnnotation(clazz, ApiExposed.class) != null;
+  public boolean isApiExposed(Class<?> clazz) {
+    ApiExposed ann = getApiExposedAnnotation(clazz);
+    return ann != null && ann.value();
+  }
+
+  public boolean isApiExposed(Method method) {
```

Comment:
include hint, that beside the method the annotation is also considered if put on the declaring class